cclib/parser/gaussianparser.py

```diff
@@ -2226,19 +2456,24 @@ def extract_charges_spins(line,prop):
                         else:
                             extract_charges_spins(line,prop)
                         
-        if line.strip() == "Natural Population":
+        if "Natural Population" in line and line.strip().split()[0] == "Natural" and line.strip().split()[1] == "Population":
             if not hasattr(self, 'atomcharges'):
                 self.atomcharges = {}
+            if not hasattr(self, 'atomspins'):
+                self.atomspins = {}
             if "natural" not in self.atomcharges:
                 line1 = next(inputfile)
                 line2 = next(inputfile)
                 if line1.split()[0] == 'Natural' and line2.split()[2] == 'Charge':
                     dashes = next(inputfile)
                     charges = []
+                    spins = []
                     for i in range(self.natom):
                         nline = next(inputfile)
                         charges.append(float(nline.split()[2]))
+                        spins.append(float(nline.split()[-1]))
                     self.atomcharges["natural"] = charges
+                    self.atomspins["natural"] = spins
```

Comment:
Yes, this is an example file of NBO with spins:

[spin_NBO.log](https://github.com/cclib/cclib/files/12497380/spin_NBO.log)

The code might have an issue though, since the atomspins attribute might be included always (but only open-shell species should have it). Please, could you take a look in your dev version? This is another NBO file where atomspins should not show up:

[closedshell_NBO.log](https://github.com/cclib/cclib/files/12497387/closedshell_NBO.log)

Reply:
@oliver-s-lee should this thread be resolved?

Reply:
Not quite matey, still working through it :)

Ok so the atom spins also show up for me in the closed shell file, that's ok it's just the header section isn't quite specific enough to only identify the open shell version. I'll fix and then commit if that's ok.

Reply:
sounds good, go for it